xtas/tasks/single.py

```diff
@@ -319,8 +319,8 @@ def semanticize(doc, lang='en'):
         raise ValueError("not a valid language: %r" % lang)
     text = fetch(doc)
     url = 'http://semanticize.uva.nl/api/%s?%s' % (lang,
-                                                   urlencode({'text': text}))
```

Comment:
Does that server have a URL length limit? Maybe POST would be better for large documents.